tests/layer_tests/common/utils/tf_utils.py

```diff
@@ -135,3 +163,49 @@ def permute_nchw_to_nhwc(shape, use_new_frontend=False):
 
 def permute_axis(axis, permutation_inv):
     return permutation_inv[axis]
+
+
+def transpose_nchw_to_nhwc(data, use_new_frontend, use_old_api):
+    if use_new_frontend or not use_old_api:
+        return data
+
+    if len(data.shape) == 4:  # reshaping for 4D tensors
+        return data.transpose(0, 2, 3, 1)
+    elif len(data.shape) == 5:  # reshaping for 5D tensors
+        return data.transpose(0, 2, 3, 4, 1)
+    else:
+        return data
+
+
+def transpose_nhwc_to_nchw(data, use_new_frontend, use_old_api):
+    if use_new_frontend or not use_old_api:
+        return data
+
+    if len(data.shape) == 4:  # reshaping for 4D tensors
+        return data.transpose(0, 3, 1, 2)  # 2, 0, 1
+    elif len(data.shape) == 5:  # reshaping for 5D tensors
+        return data.transpose(0, 4, 1, 2, 3)  # 3, 0, 1, 2
+    else:
+        return data
```

Comment:
This routines not necessarily use for TFLite only. It makes sense to put them to the common utils then.